http_check/datadog_checks/http_check/http_check.py

```diff
@@ -353,3 +350,10 @@ def check_cert_expiration(
 
         else:
             return AgentCheck.OK, days_left, seconds_left, "Days left: {}".format(days_left)
+
+    @staticmethod
+    def _include_content(include_content, message, content):
+        if include_content:
+            message += '\nContent: {}'.format(content[:MESSAGE_LENGTH])
+            message = message[:MESSAGE_LENGTH]
```

Comment:
This might be more concise/efficient ?

```suggestion
            message += '\nContent: {}'.format(content[:MESSAGE_LENGTH - len(message)])
```